### PR TITLE
Add Rainmeter

### LINKED ISSUE #3552: Icon Request: Rainmeter
**Name:** Rainmeter
**Website:** https://www.rainmeter.net/
**Alexa rank:** [42,670](https://www.alexa.com/siteinfo/rainmeter.net)
  

**Official resources for icon and color:** https://github.com/rainmeter/rainmeter-www/tree/master/source/img

### PR DESCRIPTION
**Pull request by creepertron95**
rainmeter

**Issue:** Closes #3552
**Alexa rank:** [42,670](https://www.alexa.com/siteinfo/rainmeter.net)
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
 The hex value is picked from their blue logo from the branding sources. Icon is extracted from their original svg logo.

### CODE REVIEW

**Review by runxel — CHANGES_REQUESTED** [review]
The source remains unclear, so yours looks pretty off rn.
rm

I did my comparison to the [logo_blue.ai](https://github.com/rainmeter/rainmeter-www/blob/867fd905fda8d1b1083730adcb7f49f1775cb5b0/source/img/logo_blue.ai).

**Comment by creepertron95** [comment]
@runxel I used it's svg file in the source. (https://github.com/rainmeter/rainmeter-www/blob/master/source/img/logo_nav.svg)

The color is from (https://github.com/rainmeter/rainmeter-www/blob/master/source/img/logo_blue.ai)

**Comment by runxel** [comment]
Maybe use the AI file for everything, since this seems to be the source.
Also the color in the AI is in fact `#19519b`... not `#12509C`? 🤔 Checked it twice now.

**Comment by creepertron95** [comment]
The svg was successful in the lint. @runxel

File: icons/rainmeter.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Rainmeter icon</title><path d="M12 0l-.7 1.088S9.49 3.91 7.677 7.205c-.908 1.647-1.818 3.413-2.51 5.025-.692 1.613-1.191 3.029-1.191 4.225C3.975 20.647 7.609 24 12 24s8.025-3.353 8.025-7.545c0-1.196-.499-2.612-1.191-4.225-.692-1.612-1.602-3.378-2.51-5.025-1.814-3.295-3.627-6.117-3.627-6.117L12 0zm0 3.168c.507.804 1.435 2.239 2.87 4.842.89 1.618 1.78 3.348 2.437 4.879.657 1.53 1.056 2.91 1.056 3.566 0 3.226-2.807 5.879-6.363 5.879-3.555 0-6.363-2.653-6.363-5.879 0-.657.4-2.035 1.056-3.566.657-1.531 1.547-3.261 2.438-4.88C10.564 5.408 11.493 3.973 12 3.169zM9.262 14.609c-1.533-.077-2.15 1.16-2.15 1.16a3.033 3.033 0 00-.06.55c0 2.547 2.216 4.613 4.948 4.613s4.947-2.066 4.947-4.614a3.05 3.05 0 00-.06-.548s-1.946 2.215-4.938 0c-1.104-.818-1.99-1.125-2.687-1.16z"/></svg>
```
Comment (adamrusted) [inline icons/rainmeter.svg:1]: I'm spotting some slight differences between your PR (red) and source - built from the .AI file (black).
image

Having generated the path from in Illustrator, I've been able to get it working with a precision of 3 - would you mind altering the path to this, and we can merge this PR?
```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Rainmeter icon</title><path d="M12.7 1.088L12 0l-.7 1.088c-.751 1.168-7.342 11.525-7.342 15.366C3.958 20.615 7.565 24 12 24s8.042-3.385 8.042-7.546c0-3.84-6.591-14.197-7.342-15.366zM12 22.335c-3.516 0-6.377-2.638-6.377-5.881C5.623 13.908 9.732 6.756 12 3.1c2.268 3.656 6.377 10.809 6.377 13.355 0 3.242-2.861 5.88-6.377 5.88zm4.957-6.017c0 2.548-2.22 4.615-4.957 4.615-2.737 0-4.957-2.067-4.957-4.615 0-.163.021-.347.058-.549 0 0 1.306-2.616 4.847 0 2.999 2.215 4.95 0 4.95 0 .038.202.059.386.059.549z"/></svg>
```

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by adamrusted** [comment]
@simple-icons/maintainers - just pushed the rebuilt icon to this branch.

**Comment by fbernhart** [comment]
@adamrusted LGTM 💧 

Thanks as well to @creepertron95 for the initial work on this icon. 🥇

**Pull request merged**